src/main/services/downloaders/real-debrid.downloader.ts

```diff
@@ -22,52 +26,14 @@ export class HTTPDownloader extends Downloader {
     return 1;
   }
 
-  static async getDownloadUrl(game: Game) {
```

Comment:
I have moved this method to the real debrid client, given that all it does is to turn a magnet into a downloadable url by purely calling all the real debrid methods.